### DISCUSSION TITLE
Legion Toolkit 2.20.2 - Legion 7i 16ITHG6 fan speed issue

### DISCUSSION THREAD

**Discussion by TimmyTurner51** [BartoszCichecki/LenovoLegionToolkit#1193] [Q&A]
So idk what happened exactly here other than my BIOS got reset after a brick. I reset serial number, machine model etc in BIOS to my exact machine and from Lenovo Support site as I've registered my laptop luckily. It all seems okay now there. But when I go into the toolkit and set it to custom mode, and Maximum Fan Speed is set to On, it doesn't do anything whatsoever. Any ideas? After changing BIOS settings with a BIOS programming USB I checked Legion Toolkit and it appears A-OK, LegionFanControl says something about VulnerableDriverBlocklist. As that's another program I don't think these work the same, adn I do believe that may not have anything to do with this. As a last resort I decided the main place for the project is the way to go. I am dumbfounded and out of ideas.

**Comment by BartoszCichecki** [comment]
It would be good to start with attaching a log file.
